docs/examples/index.html

```diff
@@ -72,6 +72,9 @@
     <a class="current" href="./">Examples</a>
     <ul class="subnav">
             
+    <li class="toctree-l2"><a href="#overview">Overview</a></li>
```

Comment:
Please revert this commit [5073ccf290c441197fa19e171d7b06eb391999b2](https://github.com/rhiever/tpot/pull/600/commits/5073ccf290c441197fa19e171d7b06eb391999b2) because `mkdocs` can easily generate tons of conflicts with other PRs

Reply:
Thanks for the feedback.
Does your comment imply that the documentation about [contributions](https://rhiever.github.io/tpot/contributing/#updating-the-documentation) should be updated, too, then (since I was simply following my interpretation of the instructions there)?

Reply:
Hmm, maybe that part should be discarded from contribution page. Sorry for the confusion. Should we update contributions about `mkdocs`  @rhiever? I remembered you mentioned once before.

Reply:
Yes, please update the contrib page, @weixuanfu.

Reply:
@rhiever Cool, just updated